src/ai/backend/manager/server.py

```diff
@@ -633,6 +634,13 @@ async def server_main(loop: asyncio.AbstractEventLoop,
                 str(root_ctx.local_config['manager']['ssl-cert']),
                 str(root_ctx.local_config['manager']['ssl-privkey']),
             )
+
+        # Start aiomonitor.
+        # Port is set by config (default=50001).
+        m = aiomonitor.Monitor(loop, port=root_ctx.local_config['manager']['aiomonitor-port'])
```

Comment:
It has [an extra port for telnet-based aioconsole](https://github.com/aio-libs/aiomonitor/blob/master/aiomonitor/monitor.py#L75-L76). Please disable it or provide the port number option for it as well.
Personally I'd prefer disable it since we can just use `python -m aiomonitor -p <port>` command to access the monitor console.